dist/backdraft.js

```diff
@@ -1214,42 +1257,67 @@ _.extend(Plugin.factory, {
     // @col: The column from the column manager corresponding to the column we're creating
     //   filtering wrappers for.
     _createFilteringWrappers: function(head, col) {
```

Comment:
Why does "using vies" mean it has to be done outside of Backdraft repo?  Couldn't they be backdraft views, which are pretty similar to a backbone view? 

Because I agree with Zsolt's suggestion to have the templates and event handling within views, and much less code hand-written in the the DataTable class(es)